packages/browser/src/Component/Widget/Server/SPARQL/Service/SparqlEndpointResponseProvider.js

```diff
@@ -10,7 +10,7 @@ export class SparqlEndpointResponseProvider extends EventSender {
   /**
    * Creates a SPARQL Endpoint Provider
    *
-   * @param {object} configSparql - config sparql need description
+   * @param {object} configSparql - The SPARQL Widget configuration
```

Comment:
could be nice to describe config structure with jsdoc

Reply:
this jsdoc is described by the class that uses the config :
packages/browser/src/Component/Widget/Server/SPARQL/Service/SparqlEndpointService.js